Content.Server/Content.Server.csproj

```diff
@@ -21,6 +21,7 @@
     <ProjectReference Include="..\Content.Server.Database\Content.Server.Database.csproj" />
     <ProjectReference Include="..\Content.Shared.Database\Content.Shared.Database.csproj" />
     <ProjectReference Include="..\RobustToolbox\Lidgren.Network\Lidgren.Network.csproj" />
+    <ProjectReference Include="..\RobustToolbox\Robust.Client\Robust.Client.csproj" />
```

Comment:
Why is this here though?

Reply:
i dont know, imma take that out. dunno why it keeps trying to commit stuff like this